python/lsst/sims/catUtils/mixins/sncat.py

```diff
@@ -32,6 +42,11 @@ class SNFunctionality(object):
     which define these attributes.
     """
 
+    # Write the location of SED file (for example for PhoSim)
+    writeSedFile = False
+    # prefix to use for SED File name
+    prefix = ''
```

Comment:
'prefix' is too generic a name.  It will get used for something else.  Can this be `sn_sed_file_prefix` or something equally specific?